packages/common/src/utils/conditionalFormatting.ts

```diff
@@ -111,6 +112,17 @@ export const getMinMaxFromMinMaxMap = (
     max: Math.max(...Object.values(minMaxMap).map((m) => m.max)),
 });
 
+export class ConditionalFormattingError extends LightdashError {
+    constructor(message: string) {
+        super({
+            message,
+            name: 'ConditionalFormattingError',
+            statusCode: 400,
+            data: {},
+        });
+    }
+}
```

Comment:
this error is very specific and I think it's better to leave it in this file

Reply:
nice!!